sap-s4hana-cloud/2.6/modules/ROOT/pages/sap-s4hana-cloud-connector-reference.adoc

```diff
@@ -0,0 +1,1412 @@
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Client timeout unit for the request |  SECONDS |
+| Retrieve Csrf Token a| Boolean |  Enables or disables automatic X-CSRF-Token retrieval before a resource-modifying request, such as a POST request. |  true |
+| TLS configuration a| <<Tls>> | Defines a configuration for TLS, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol then the user needs to configure at least the keystore in the `tls:context` child element of the `listener-config`. | `HTTP` |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+
+[[Config_Oauth2ClientCredentialsAuthentication]]
+=== OAuth 2.0 Client Credentials Authentication
+
+include::connectors::partial$oauth2-description.adoc[]
+
+==== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Service base URL a| String |  The base URL of the SAP S/4HANA instance to access from the connector and from which all other endpoints are derived. |  | x
+| Cache Entry TTL a| Number |  The time to live. The time a cache entry is kept in the cache before being evicted. A value of 0 disables the cache. |  0 |
+| Cache Entry TTL Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *Cache Entry TTL* field. |  MINUTES |
+| EDM Expiration Interval a| Number |  The interval to check for expired EDM objects in the ObjectStore. |  30 |
+| EDM Expiration Interval Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *EDM Expiration Interval* field. |  MINUTES |
+| Discovery Service Enabled a| Boolean |  Feature switch for automatic service discovery. |  `true` |
+| Scoped service discovery relative path a| String | Relative path from the base URL to the endpoint used for automatic services discovery with custom namespaces on cloud instances. |  `sap/opu/odata/IWFND/CATALOGSERVICE;v=0002/ScopedServiceCollection` | 
+| Unlisted services directory a| String |  Common HTTP directory for unlisted services and services declared in *Service Attribute* (if the service URL is not specified). This parameter enables users to specify only service names in the unlisted services list instead of service URLs or enables users to not specify a service URL in *Services Attribute*. It also enables the connector to construct the service URL using the following format: `{base URL parameter}/{this parameter}/{item from unlisted services list or service name from *Service Attribute*}` |   |
```

Comment:
```suggestion
| Unlisted services directory a| String |  Common HTTP directory for unlisted services and services declared in *Service Attribute* (if the service URL is not specified). This parameter enables users to specify only service names in the unlisted services list instead of service URLs or enables users to not specify a service URL in *Services Attribute*. It also enables the connector to construct the service URL using the following format: `{base URL parameter}/{this parameter}/{item from unlisted services list or service name from *Service Attribute*}`. |   |
```